lib/sisimai/datetime.rb

```diff
@@ -379,10 +379,7 @@ def parse(argv1)
                 # ISO 8601; 2000-04-29T01:23:45
                 v[:Y] = cr[1].to_i
                 v[:M] = MonthName[:abbr][cr[2].to_i - 1]
-
-                if cr[3].to_i < 32
-                  v[:d] = cr[3].to_i
-                end
+                v[:d] = cr[3].to_i if cr[3].to_i < 32
```

Comment:
__[rubocop]__
* Metrics/BlockNesting: Avoid more than 4 levels of block nesting. [:ref](https://github.com/bbatsov/ruby-style-guide#three-is-the-number-thou-shalt-count)

<!-- Comment by [SideCI](https://sideci.com) -->